templates/CRM/Admin/Page/LocationType.tpl

```diff
@@ -72,4 +73,5 @@
     {crmButton q="action=add&reset=1" id="newLocationType" icon="plus-circle"}{ts}Add Option{/ts}{/crmButton}
     {crmButton p="civicrm/admin" q="reset=1" class="cancel" icon="times"}{ts}Done{/ts}{/crmButton}
   </div>
+</div>
```

Comment:
How is this file related to field name change?

Reply:
@pradpnayak This is not just for adding label. PR is to make the structure proper and this file adds missing wrapper around the content like other files.

Reply:
:)

Reply:
Should I understand :) as approved then?

Reply:
Yes APPROVED.

Reply:
@pradpnayak can you do that from above Review changes then? for others to know the status